vcl_templates/www.vcl.erb

```diff
@@ -220,6 +220,10 @@ sub vcl_recv {
     set req.http.GOVUK-ABTest-EducationNavigation = "A";
   } else if (req.url ~ "[\?\&]ABTest-EducationNavigation=B(&|$)") {
     set req.http.GOVUK-ABTest-EducationNavigation = "B";
+  } else if (req.url ~ "^\/education(\/)?.*$") {
```

Comment:
Yup, I can simplify to `^/education` without the slash so it works on the root taxon page.